### DISCUSSION TITLE
unable to generate full course

### DISCUSSION THREAD

**Discussion by AndyH1965** [Courseplay/Courseplay_FS22#1074] [Q&A] [answered]
Hi, I have created a merged field with a lot of obstacles/islands, these are a mix of telegraph poles and trees, when i try to generate a course for a convoy of 2 seeders Course play takes roughly 40 minutes and can't seem to generate a full course is this field just too complicated, image attached is the generated course, also when i try to generate a couse for a single seeder it only does one headland and the course stops there
![think i broke course play](https://user-images.githubusercontent.com/100184107/155068359-1ec71fe3-aeee-4852-8f2b-96acc4816c25.jpg)

**Comment by Tensuko** [comment]
Sry but thies field is some real struggle.... It is a wonder that it generated it so far already...
Plus Multitool Course, that won't work well anyway...

**Comment by pvaiko** [comment, marked as answer]
Sorry, you have to simplify that, split it in two at least, not sure if any IRL farmer would want to work on such a convoluted field anyway.

Generating a usable course for that field would probably worth a PhD.

**Reply by AndyH1965** [reply]
![convoy seeding](https://user-images.githubusercontent.com/100184107/155226152-34ed072b-d421-4b36-ae6e-f6515c623149.jpg)
i found the issue was with the number of islands so i removed some in giants editor and tried again this time it generated a course for 2 seed hawk seeders in convoy, this course play is amazing , thank you for answering too :)
